### ISSUE TITLE
Title of project overlaps photo in slider when window is a bit minimised

### ISSUE THREAD

**Issue by elliec** [geotagx/geotagx-theme#46]

**Comment by spMohanty** [comment]
@supranove : This looks like a minor CSS issue, can you look at it when you have time ?

**Comment by othieno** [comment]
@spMohanty will do!

**Comment by elliec** [comment]
@spMohanty @supranove was this fixed ?

**Comment by othieno** [comment]
@elliec No, this is still not solved. We'll either have to change the slider we currently use or modify it to account for small screens. It's pretty low-priority for now.

**Comment by elliec** [comment]
@supranove is this one fixed with your commit?

**Comment by othieno** [comment]
@elliec so it's been fixed but it's currently on a separate, experimental branch. No merges have been made yet because a few things are broken (uhmm, experimental) but the issue has been resolved. I'll leave this open as a reminder, and close it once the changes have been finalized.
